Source/Core/Common/Assembler/GekkoLexer.cpp

```diff
@@ -649,44 +654,39 @@ TokenType Lexer::ClassifyAlnum() const
 
     return false;
   };
-  constexpr auto eq_nocase = [](std::string_view str, std::string_view lwr) {
-    auto it_l = str.cbegin(), it_r = lwr.cbegin();
-    for (; it_l != str.cend() && it_r != lwr.cend(); it_l++, it_r++)
-    {
-      if (std::tolower(*it_l) != *it_r)
-      {
-        return false;
-      }
-    }
-    return it_l == str.end() && it_r == lwr.end();
-  };
 
   if (std::tolower(alnum[0]) == 'r' && valid_regnum(alnum.substr(1)))
   {
     return TokenType::GPR;
   }
+  
```

Comment:
The extra newline here before the `else if` is bugging me. I'm surprised our clang-format doesn't remove those.

Reply:
Unintentional, it can be removed if you'd like